src/Sylius/Bundle/AdminBundle/Console/Command/DeleteAdminUserCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Console\Command;
+
+use Sylius\Component\User\Model\UserInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+
+#[AsCommand(
+    name: 'sylius:admin-user:delete',
+    description: 'Deletes an admin user account by the given username',
+)]
+final class DeleteAdminUserCommand extends Command
+{
+    protected SymfonyStyle $io;
+
+    /** @param UserRepositoryInterface<UserInterface> $adminUserRepository */
+    public function __construct(
+        private readonly UserRepositoryInterface $adminUserRepository
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (!$input->isInteractive()) {
+            $this->io->error('This command must be run interactively.');
+            return Command::FAILURE;
+        }
```

Comment:
I'd be for making it viable with non-interactive as well, just pass an optional argument that'll identify the admin when it is

Reply:
> I'd be for making it viable with non-interactive as well, just pass an optional argument that'll identify the admin when it is

Mhh. What is the difference between non-interactive and interactive. And how should the command be used then ? Tbh when working on this i wasnt sure if this would be needed 😕

Reply:
With non-interactive you could write
> sylius:admin-user:delete test@example.com --no-interaction

and the user either would be gone or you'd get a notice that it doesn't exist.
I wouldn't call it a requirement, but it's a nice thing to have.

Reply:
And the method to check in the if would be also `input->isInteractive()`?

Reply:
Yes, it's the same method

Reply:
![SCR-20240308-pjye](https://github.com/Sylius/Sylius/assets/39345336/fb129981-c87d-4c98-a993-516c1a965111)

Hm. Doesent really understand why it says `$input->isInteractive()` is always true. seems `true` is the default 🤔